app/src/main/java/org/wikipedia/history/db/HistoryEntryWithImage.kt

```diff
@@ -0,0 +1,16 @@
+package org.wikipedia.history.db
+
+import java.util.*
+
+data class HistoryEntryWithImage(
+    val id: Int,
```

Comment:
Where is this `id` being used? Cannot find the usage of it.

Reply:
Correct.